config/config.exs

```diff
@@ -82,6 +82,10 @@ config :trento, :pow,
 # Agent heartbeat interval. Adding one extra second to the agent 5s interval to avoid glitches
 config :trento, Trento.Heartbeats, interval: :timer.seconds(6)
 
+# This is passed to the frontend as the time after the last heartbeat
+# to wait before displaying the deregistration button
+config :trento, deregistration_debounce: :timer.seconds(0)
```

Comment:
Why this is zero? I don't think this a good value, but anyway this is not really used in this pr other then passed to the frontend, which is a no-op without the frontend, pls remove it and we'll discuss this when we will make the frontend.

I know that we add it in the middleware but right know is zero and this a no-op, let's rediscuss it on the frontend

Reply:
This was previously discussed to be set to 0, but as a configurable value.
We do still need it for this PR, it's used in the [middleware](https://github.com/trento-project/web/pull/1450/files/06491f0da5b7c449b717342efa62f4d3ec0eb43c#diff-5513fdb4d4dc67a57b45ef3b29c62825cd3c9a11f3ca22c5485067c9d0945e9dR13) in order to validate the timing on the backend.

Reply:
@CDimonaco It is actually used. With the value set as zero. This is not a frontend operation, it simply makes the backend more robust against the usage of this command. That the frontend could use this value for its own work doesn't make it a no-op.